console.rst

```diff
@@ -171,6 +171,12 @@ You can optionally define a description, help message and the
     classes, but it won't show any description for commands that use the
     ``setDescription()`` method instead of the static property.
 
+.. deprecated:: 6.1
+
+    The static property ``$defaultDescription`` was deprecated in Symfony 6.1.
+    Declare your command description with the ``#[AsCommand]`` attribute
```

Comment:
```suggestion
    Declare your optional command description with the ``#[AsCommand]`` attribute
```

Reply:
And please end this sentence with a dot 👍

Reply:
Updated while merging. Thanks.